### PR TITLE
Login for mobile Styling changes

### PR DESCRIPTION
**Pull request by MauroGuerrero20**
- Styling changes for ```Login.jsx```.
  - Better UI for mobile when Logging in.

# Demo
Screenshot 2020-08-05 at 5 52 35 PM

### CODE REVIEW

**Review by steven-solar — APPROVED** [review]
Looks great!

**Review by antoniolinhart — APPROVED** [review]
Looks great on mobile!

File: capstone-frontend/src/styles/Login.css [hunk 1]
```diff
@@ -10,4 +10,40 @@
...
 #sign-in-text button{
   font-size: 18px !important;
+}
+
+/* Tablet */
+@media only screen and (max-width: 768px) {
+  #login-card {
+    margin: auto;
+    transform: translateY(50%);
+    width: 35rem;
```
Comment (geoffbass) [inline capstone-frontend/src/styles/Login.css:20]: nice use of rems!

**Review by geoffbass — APPROVED** [review]
LGTM

**Pull request merged**